Angular/Angular14/protected/jsdefender.config.json

```diff
@@ -25,10 +25,8 @@
             "lower": 0,
             "upper": 65535
         },
-        "localDeclarations": {
-            "nameMangling": "base52"
-        },
-        "propertyIndirection": true,
+        "localDeclarations": false,
+        "propertyIndirection": false,
```

Comment:
is there a reason we turned these two off?

Reply:
Yeah, so local declarations couldn't be enabled, because there are some eval's in the Angular 14 code and it wouldn't work properly with JSDefender. PropertyIndirection was disabled, because when it's enabled, the sample would just throw some JS errors

Reply:
I see, good